### ISSUE TITLE
TypeError: moments() got an unexpected keyword argument 'shift'

### ISSUE THREAD

**Issue by Esiravegna** [fchollet/keras#5163]
Hey there. When trying to import InceptionV3, ResNet50, or Xception, using 1.2.1 I got this error:

```python
>>> from keras.applications.xception import Xception
>>> m = Xception()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/mnt/hd/envs/esteban/miniconda3/envs/caffe/lib/python2.7/site-packages/keras/applications/xception.py", line 114, in Xception
    x = BatchNormalization(name='block1_conv1_bn')(x)
  File "/mnt/hd/envs/esteban/miniconda3/envs/caffe/lib/python2.7/site-packages/keras/engine/topology.py", line 572, in __call__
    self.add_inbound_node(inbound_layers, node_indices, tensor_indices)
  File "/mnt/hd/envs/esteban/miniconda3/envs/caffe/lib/python2.7/site-packages/keras/engine/topology.py", line 635, in add_inbound_node
    Node.create_node(self, inbound_layers, node_indices, tensor_indices)
  File "/mnt/hd/envs/esteban/miniconda3/envs/caffe/lib/python2.7/site-packages/keras/engine/topology.py", line 166, in create_node
    output_tensors = to_list(outbound_layer.call(input_tensors[0], mask=input_masks[0]))
  File "/mnt/hd/envs/esteban/miniconda3/envs/caffe/lib/python2.7/site-packages/keras/layers/normalization.py", line 120, in call
    epsilon=self.epsilon)
  File "/mnt/hd/envs/esteban/miniconda3/envs/caffe/lib/python2.7/site-packages/keras/backend/tensorflow_backend.py", line 1375, in normalize_batch_in_training
    shift=None, name=None, keep_dims=False)
TypeError: moments() got an unexpected keyword argument 'shift'
````

VGG16 runs just fine, tough.
Anything on my end that I could do? Thanks!

**Comment by bstriner** [comment]
Are you on latest tensorflow and keras? What is your setup?

Works fine for me on bleeding-edge keras/tf/ubuntu.

Cheers

**Comment by bstriner** [comment]
Tensorflow, theano and keras change so quickly that I would recommend using the bleeding-edge at all times if possible.

**Comment by patyork** [comment]
What version of TF are you using? `shift` has been supported since v.10 (or May 2016)

**Comment by fchollet** [comment]
The fix is to upgrade your TF version to 0.12.
